### PR TITLE
JAV-470 resolve enum param validate fail has no response

### PR DESCRIPTION
**Pull request by jeho0815**

### CODE REVIEW

**Comment by liubao68** [comment]
What's the difference with https://github.com/ServiceComb/ServiceComb-Java-Chassis/pull/211? It seems a wrong patch.

File: swagger/swagger-invocation/invocation-core/src/main/java/io/servicecomb/swagger/invocation/arguments/ArgumentsMapperFactory.java [hunk 1]
```diff
@@ -99,10 +99,12 @@ protected boolean isSwaggerWrapBody(ArgumentsMapperConfig config, List<ProviderP
       return false;
     }
 
-    swaggerType = ((Class<?>) swaggerType).getFields()[0].getGenericType();
-    Converter converter = converterMgr.findConverter(type, firstProviderParam, swaggerType);
-    if (ConverterCommon.class.isInstance(converter)) {
-      return false;
+    if (((Class<?>) swaggerType).getFields().length > 0) {
```
Comment (wujimin) [inline swagger/swagger-invocation/invocation-core/src/main/java/io/servicecomb/swagger/invocation/arguments/ArgumentsMapperFactory.java]: must have a field in this place
if there is no field, that means it's bug, MUST not think this is a normal scenes.

Comment (jeho0815) [reply swagger/swagger-invocation/invocation-core/src/main/java/io/servicecomb/swagger/invocation/arguments/ArgumentsMapperFactory.java]: has remove this modify

**Comment by jeho0815** [comment]
@liubao68 this is resolve exception in method runOnExecutor() has no response

File: common/common-rest/src/test/java/io/servicecomb/common/rest/TestRestProducerInvocation.java [hunk 2]
```diff
@@ -202,6 +202,29 @@ void invoke() {
...
+      @Mock
+      void invoke() {
+        invokeNoParam = true;
+      }
+    }.getMockInstance();
+    initRestProducerInvocation();
+    try {
+      restProducerInvocation.runOnExecutor();
+    } catch (Exception e) {
+      e.printStackTrace();
```
Comment (WillemJiang) [inline common/common-rest/src/test/java/io/servicecomb/common/rest/TestRestProducerInvocation.java]: If the exception is thrown, you need to write assert for the exception and don't just print it out.

Comment (liubao68) [reply common/common-rest/src/test/java/io/servicecomb/common/rest/TestRestProducerInvocation.java]: Assert is done above the expectation, method invoke is not executed and result is false. I think it's his logic.

Comment (jeho0815) [reply common/common-rest/src/test/java/io/servicecomb/common/rest/TestRestProducerInvocation.java]: yes, this is simulate a exception

Comment (wujimin) [reply common/common-rest/src/test/java/io/servicecomb/common/rest/TestRestProducerInvocation.java]: best practice is use org.junit.rules.ExpectedException

**Comment by jeho0815** [comment]
use another pr trace

**Pull request closed without merging**